test/org/loklak/api/search/GithubProfileScraperTest.java

```diff
@@ -46,11 +46,11 @@ public void githubProfileScraperOrgTest() {
 
         Post fetchedProfile = githubScraper.scrapeGithub(profile, br);
 
-        assertEquals(fetchedProfile.getString("short_description"), shortDescription);
-        assertEquals(fetchedProfile.getString("user"), userName);
-        assertEquals(fetchedProfile.getString("user_id"), userId);
-        assertEquals(fetchedProfile.getString("location"), location);
-        assertEquals(fetchedProfile.getString("special_link"), specialLink);
```

Comment:
see here, there are some indentation convention to leave a space after comma. Could you add the space after comma in new changes please!!